src/components/shared/Layout.tsx

```diff
@@ -15,6 +15,11 @@ export interface SceneProps {
   expo: Exposition[];
   image: string;
   imageAlt?: string;
+  style?: {
+    width?: string,
+    margin?: string,
+    border?: string,
+  };
```

Comment:
i think we can just do smth like `CSS.Properties`

Reply:
Ok I will take a stab